CS-458/academic-requirements
# Account/schedule list

# Jira Ticket

Jira ticket number with link:

# Description of Changes
A Navigation Menu would be added to the Main branch, which would allow for easy connection to other web pages. In turn, a basic Account page has been added, with its respective navigation menu holding full functionality. 

# Type of Change

- [  ] Feature Update

# How has this been tested?
_Please describe the tests that you used to verify your changes._

**Automated Tests (Preferred)**

**Manual Tests**
Manually tested by clicking on the various links in the navigation menu to see if they properly connect to each different web page. 

# Checklist
- [x] My code follows ES Lint guidelines
- [x] My solution meets all of the acceptance criteria for the user story
- [ ] My code contains necessary comments and documentation
- [ ] I have added tests that are useful and cover the new changes
